hack/make-apiserver-certs.sh

```diff
@@ -1,30 +1,31 @@
 #!/usr/bin/env bash
+set -e
 
 export SVC_NAME="${SVC_NAME:-hiveapi}"
 export SVC_NAMESPACE="${SVC_NAMESPACE:-hive}"
 
-mkdir -p "config/apiserver/certificates"
+mkdir "hiveapi-certs"
```

Comment:
I am not sure why these needed to move. The config/apiserver/certificates folder was already being ignored by git. And .gitignore has not been updated to ignore the new hiveapi-certs folder.

Reply:
ignored by git, but hack/update-bindata.sh was picking up those files (mostly just trying to follow what we've done for hiveadmission files)

Reply:
Ah, OK.